ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  ansible:
+    name: "4.11"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.11"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
```

Comment:
If the test aims at 4.10 to 4.11 upgrade, then we don't need the release.

Reply:
Per the previous discussion in https://github.com/openshift/release/pull/36814#discussion_r1130589518, I guess Mike want to ensure the ARO cluster is upgraded to the latest z-stream version before upgrade to the next major release.

Reply:
Good to know! The thing is if we use stable as `latest`, it will get the latest z version by default. We don't need to do z version upgrade any more.

Reply:
Per Mike's feedback, now ARO have a obvious limitation, users can not specify ARO cluster version on fresh install, but determinated by az cli. We keep the `latest` build, is because of some dependency referenced steps mentioned in https://github.com/openshift/release/pull/36814#issuecomment-1468547742. Actually the `latest` build does not control the source version of ARO cluster. 

Of course, if `intermediate410` and `latest` is pointing to the same stable build, I think we can remove the `intermediate410` one, use `latest` to replace it in upgrade path.

Reply:
It might be better to leave it as is then since I want to upgrade the cluster to the newest supported 4.10 release before proceeding further.  I can't set release:latest to anything other than ci or nightly per : 

https://github.com/openshift/release/pull/36814#issuecomment-1468547742

Reply:
Got it.

Reply:
Ack. Apologize for misleading you. FYI: in addition to `stable` channel, `fast` channel also holds officially supported release. And a release goes to `fast` channel earlier than `stable` channel. For instance, the latest version in `stable-4.10` is 4.10.53, while it's 4.10.54 in `fast-4.10`.